scripts/styles_check.sh

```diff
@@ -3,14 +3,14 @@
 # This script is used to check if the style.css file is up to date.
 
 # Check if the git tree for CWD is clean
-if [ -n "$(git status --porcelain)" ]; then
+if [ -n "$(git status internal/view/assets/less internal/view/assets/css --porcelain)" ]; then
     echo "❌ git tree is not clean. Please commit all changes before running this script."
     exit 1
 fi
 
 # Check style.css file
 CLEANCSS_OPTS=${CLEANCSS_OPTS} make styles
-if [ -n "$(git status --porcelain)" ]; then
+if [ -n "$(git status internal/view/assets/less internal/view/assets/css --porcelain)" ]; then
     echo "❌ style.css wasn't built from less changes. Please run 'make styles' and commit the changes."
     git reset --hard
```

Comment:
This would reset all repository to the previous state, instead of only the generated files, I think using `git checkout` here is better now.